core/src/test/java/com/linecorp/armeria/server/ServerTest.java

```diff
@@ -436,6 +436,18 @@ public void notGracefulShutdownBlockingTaskExecutor() {
         assertThat(MoreExecutors.shutdownAndAwaitTermination(executor, 10, TimeUnit.SECONDS)).isTrue();
     }
 
+    @Test
+    public void versionMetrics() {
+        final Server server = ServerTest.server.server();
+        server.setupVersionMetrics();
+
+        final MeterRegistry meterRegistry = server.config().meterRegistry();
+        assertThat(meterRegistry.find("armeria.build.info")
+                                .gauge()
+                                .value())
```

Comment:
How about also checking the three tags exist (though not checking their values which isn't constant)